CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.8.3](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/compare/3.8.2...3.8.3) - 2019-02-28
+### Fixed
+- Deprecations about core bundle forms
+- Exception on `StringFilter` with null values
+- `more` in `get-autocomplete-item` always returns `true`
```

Comment:
I tried to create a changelog for #909 (fixes #908) but I cannot say I understood what this is about. @ejkun @Selion05 please advise

Reply:
Took me a while, but I figured it out. Ceil function returns float, so lastPage would always be float. But the current page is int. So the function isLastPage ([here](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Datagrid/Pager.php#L418)) would always return false. Which would make `more` in [retrieveAutocompleteItemsAction](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Action/RetrieveAutocompleteItemsAction.php#L171) to always be true.

This happens because `42.0 === 42` returns true.

Can you put this comment on his PR? To help others understand in the future.

Reply:
```
>>> 42.0 === 42
=> false
```

Reply:
Yes, sorry.